streaming/store.go

```diff
@@ -127,14 +127,15 @@ func writeJobDetails(ctx context.Context, migrationJobId string, isShardedMigrat
 		logger.Log.Error(fmt.Sprintf("internal error occurred while persisting metadata for migration job %s: %v\n", migrationJobId, err))
 		return err
 	}
-	jobDetails := SmtJobs{
+	jobDetails := SmtJob{
 		JobId:               migrationJobId,
 		JobName:             migrationJobId,
 		JobType:             constants.MINIMAL_DOWNTIME_MIGRATION,
 		Dialect:             conv.SpDialect,
+		JobStateData:        "{\"state\": \"RUNNING\"}",
```

Comment:
Weren't `CREATING` and `CREATED` the only states we were using right now? 
Wouldn't this mean all the jobs would be in `RUNNING` state forever?

Reply:
For resources, we have created and Created, deleting deleted.
for jobs, creating->running->pausing->paused->resuming->running-> Deleting->deleted.

Reply:
Since the concept of jobs is not fully introduced to fwd flows, we can keep some temporary state for them. AFAIK its not being used anywhere else its just a placeholder.